pkg/domain/events.go

```diff
@@ -10,6 +10,16 @@ type Event interface {
 	Message() string
 }
 
+type BlueprintDryRunEvent struct{}
+
+func (b BlueprintDryRunEvent) Name() string {
+	return "BlueprintDryRun"
+}
+
+func (b BlueprintDryRunEvent) Message() string {
+	return fmt.Sprintf("Executing blueprint in dry run mode")
```

Comment:
🚫 **[checkstyle]** <gosimple> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>S1039: unnecessary use of fmt.Sprintf